server/datastore/mysql/apple_mdm.go

```diff
@@ -743,20 +743,37 @@ func ingestMDMAppleDeviceFromCheckinDB(
 	}
 }
 
+func mdmHostEnrollFields(mdmHost *fleet.Host) (refetchRequested bool, lastEnrolledAt time.Time) {
+	supportsOsquery := mdmHost.SupportsOsquery()
+	// 2000-01-01 00:00:00 is what Fleet considers the zero/"Never" time.
+	lastEnrolledAt, err := time.Parse("2006-01-02 15:04:05", "2000-01-01 00:00:00")
```

Comment:
Not critical (don't block on this), but since you provide a hard-coded value, it could be constructed in a more straightforward way without error possible with `time.Date` https://pkg.go.dev/time#Date.

Reply:
(I think it's just code you refactored from before, so all the more reason to not worry about it!)